tools/cawlign/compile.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Remove extra space in front of Emscripten "-s" variables to avoid errors
+emcmake cmake -DCMAKE_EXE_LINKER_FLAGS="${EM_FLAGS//-s /-s} --preload-file test@/cawlign --preload-file res@/cawlign" .
```

Comment:
Is the `test/` folder needed for all executions of cawlign? I would lean towards not including it since it's 13 MB, which is a lot for each user to download that much.

Reply:
That's a great point! It only needs this file: `HIV1-pol-326.fa`

I'm not sure how to edit this line to only preload that single file

Reply:
I think we can remove the preloading of the whole folder and then mount the data from GitHub in the HTML. I'll give it a try now just to make sure that actually works :)